ilastik/experimental/api/_pipelines.py

```diff
@@ -52,6 +52,21 @@ class PixelClassificationPipeline:
     prob_maps = pipeline.get_probabilities(img)
     ```
 
+    Example using an OME-Zarr dataset stored remotely:
+    ```Python
+    import xarray
+    from ilastik.experimental.api import PixelClassificationPipeline
+    from lazyflow.utility.io_util.OMEZarrStore import OMEZarrStore
+
+    store = OMEZarrStore("<https://example.com/data.zarr>")
```

Comment:
This is of course very subjective, but some people could interpret `<` and `>` in example strings as literal, required characters (maybe some formatting delimiters like we have in dataset nicknames).

Reply:
I see the point and was about to change it. Then saw that all other examples in the API docstrings use the same notation, which must be why I used it. So I'll leave it :)